auth-web/src/components/auth/common/AccountSuspendAlert.vue

```diff
@@ -75,7 +80,14 @@ export default class AccountSuspendAlert extends Vue {
   }
 
   get isSuspendedForNSF (): boolean {
-    return this.currentOrganization?.statusCode === AccountStatus.NSF_SUSPENDED
+    console.log(this.currentOrganization)
+    return (this.currentOrganization?.statusCode === AccountStatus.NSF_SUSPENDED &&
+      this.currentOrganization?.suspensionReasonCode !== SuspensionReasonCode.OVERDUE_EFT)
+  }
+
+  get isSuspendedForEFTOverdue (): boolean {
+    return (this.currentOrganization?.statusCode === AccountStatus.SUSPENDED &&
```

Comment:
we're not having a separate status code for suspended, going to reuse the same one. Otherwise partners have to add it to all of their apps

probably will add in `payment_method` to `cfs_accounts` instead

Reply:
I don't really see much value in creating `SUSPENDED` vs `NSF_SUSPENDED`, they should be one in the same..

Reply:
Good find with the `suspensionReasonCode` though..

Reply:
does it has other reason that make it not NSF_SUSPENDED? I added an extra suspension reason to suspensionReasonCode

Reply:
Yup it's perfect

Reply:
suspensionReasonCode isn't set on PROD, looks like it's all blank. We'll need a ticket to fill these fields in.

Reply:
yeah I agree, we need another ticket for it.

Reply:
could you make a couple? maybe reference this PR?

Reply:
https://github.com/bcgov/entity/issues/21204

Reply:
https://github.com/bcgov/entity/issues/20087